src/hydrateWorker.js

```diff
@@ -0,0 +1,80 @@
+const { workerData, parentPort } = require("worker_threads");
+const fs = require("fs");
+const util = require("util");
+const readFile = util.promisify(fs.readFile);
+const writeFile = util.promisify(fs.writeFile);
+const { files, workerId, pkg, renderToStringOptions } = workerData;
```

Comment:
```suggestion
const { files, pkg, renderToStringOptions } = workerData;
```

`workerId` is an unused variable because the statement below are commented out